blackboxtest/nightwatch-tests/homepage.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Idea for test: Load a specific permlink and verify that the post content and a couple of comments renders correctly

Reply:
put test ideas here https://github.com/steemit/condenser/issues/2637